docs/source/reference/conditions.rst

```diff
@@ -162,23 +162,36 @@ If you are unsure how to use these object functions, check out the :ref:`conditi
 
 .. jcoad:condition:: beaten
 
-    The number of times the player has beaten a trainer.
+    The number of times the player has beaten a trainer. Saves to the specific tile.
 
 .. jcoad:condition:: beaten[x,y]
 
-    ???
+    Checks the beaten value at designated x,y coordinates.
 
 .. jcoad:condition:: battlewon
 
```

Comment:
Examples needs to go right below the .`. jcoad:[type]` line, which is before the description text. See https://github.com/jackson-nestelroad/pokengine-jcoad-docs/blob/main/docs/source/reference/types.rst for an example.